index.bs

```diff
@@ -972,80 +959,38 @@ ModuleStatus instances are initially created with the internal slots described i
   1. Let _hookResult_ be the result of promise-calling _hook_(_entry_, _source_).
   1. Return the result of transforming _hookResult_ with a fulfillment handler that, when called with argument _optionalInstance_, runs the following steps:
     1. Perform ? ExtractDependencies(_entry_, _optionalInstance_, _source_).
-    1. Perform UpgradeToStage(_entry_, "satisfy").
-    1. Return _optionalInstance_.
+    1. Return the result of transforming RequestSatisfy(_entry_, _buckle_) with a fulfillment handler that, when called, runs the following steps:
+      1. Return _optionalInstance_.
 1. Let _pCatch_ be the result of transforming _p_ with a rejection handler that, when called, runs the following steps:
   1. Set _entry_.[[Error]] to *true*.
 1. Set _instantiateStageEntry_.[[Result]] to _p_.
 1. Return _p_.
 </emu-alg>
 
-<h4 id="request-satisfy" aoid="RequestSatisfy">RequestSatisfy(entry)</h4>
+<h4 id="request-satisfy" aoid="RequestSatisfy">RequestSatisfy(entry, buckle)</h4>
```

Comment:
we might want to change the name of this abstract operation, because it is not really a "Request" anymore.